psl/psl/tests/validation/attributes/index/missing_index_warning/relation_mode_prisma/one_field_at_unique.prisma

```diff
@@ -0,0 +1,25 @@
+// no relation index validation warning on relationMode = "prisma" when a referenced field is already @unique.
+
+generator client {
+    provider        = "prisma-client-js"
+    previewFeatures = ["referentialIntegrity"]
+}
+
+datasource db {
+    provider = "mysql"
+    url = env("TEST_DATABASE_URL")
+    relationMode = "prisma"
+}
+
+model SomeUser {
+    id Int @id
+    profile Profile?
+}
+
+model Profile {
+    id Int @id
+    userId Int
+    user SomeUser @relation(fields: [userId], references: [id])
+
+    @@unique([userId])
+}
```

Comment:
I think this file is identical to the previous one.

Reply:
Thanks for spotting it